### DISCUSSION TITLE
OpenChrom Tutorial / Webinar ?

### DISCUSSION THREAD

**Discussion by Selemza** [OpenChrom/openchrom#471] [Ideas]
Please, can anyone point to a quick up-to-date tutorial for getting started with the software OpenChrom? There is some info online, but nothing actualized. I would like to promote the use of the software in my lab but it's hard to convince future users without any instructions available. Thanks

**Comment by Mailaender** [comment]
Yes, please https://lablicate.com/about/contact us.
